04_numeronym/jessodri/main_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("[a11y K8s abc]\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("expected %v but got %v", expected, actual)
+	}
+}
+
+func TestSortLetters(t *testing.T) {
+	testCases := []struct {
+		description string
+		word        string
+		expected    []string
+	}{
+		{"happy path", "onomatopoeia", []string{"o10a"}},
+		{"3 letter word", "cat", []string{"cat"}},
+		{"empty", "", []string{""}},
+	}
+
+	for _, tc := range testCases {
+		expected := tc.expected
+		actual := numeronyms(tc.word)
+		t.Run(tc.description, func(t *testing.T) {
+			assert.Equal(t, actual, expected, "got %v but wanted %v", actual, expected)
+		})
+	}
+
+	t.Run("test with multiple strings passed in", func(t *testing.T) {
```

Comment:
Consider splitting this into a seperate test function or include in examples list above